### DISCUSSION TITLE
What branch for L9?

### DISCUSSION THREAD

**Discussion by tomvo** [denniseilander/laravel-passport-scopes-restriction#25] [Q&A]
I just tried to install your package in an Laravel 9 environment but i'm getting compatibilty issues. Is there a tag I should use for L9 support?

Thanks

**Comment by denniseilander** [comment]
Hi @tomvo, 1.0.0 should work with Laravel 9

**Comment by tomvo** [comment]
hey @denniseilander thanks for your feedback. Seems to work but unfortunately we're also running on passport 11. Would it not be possible to support multiple laravel versions in the 2.0 branch?

**Comment by denniseilander** [comment]
Hi @tomvo, thanks for the extra information. I've opened a [PR](https://github.com/denniseilander/laravel-passport-scopes-restriction/pull/26) with support for Laravel 9. Would you be able to test this branch and check if it fixes your problem?

Thanks! :)

**Comment by tomvo** [comment]
hey @denniseilander this works brilliantly, thanks :-)

**Reply by denniseilander** [reply]
Hi @tomvo, nice! Thanks for testing, I'll merge the PR :)
